gnosis/safe-react-apps
# [Compound] Improve UI/UX

## What it solves
Resolves #224 

## How this PR fixes it
By adding a Tabs component to the Compound UI and allowing to supply or withdraw with the same button showing the user the necessary enable/disable statuses and colors for making the UX better

## How to test it
1) Open Compound app
2) Select supply tab, enter a quantity and click the button for start the supply process 
3) Select withdraw, enter a quantity  and click the button for start the withdrawal process
4) Verify in the Compound app all is working as expected
5) Use max button both for supply and withdraw
